### DISCUSSION TITLE
Spectral resampling (to Sentinel-2) error: "failed to open spectral library: 'NoneType' object has no attribute 'source'"

### DISCUSSION THREAD

**Discussion by MaximeTroiani** [EnMAP-Box/enmap-box#777] [Q&A]
Hi, 

I probably miss something but can not manage to perform a spectral resampling (EnMAP to Sentinel-2A). I am using EnMAP-Box 3.12.3 running on QGIS 3.34.2. 

The issue seems to be related to the opening of the spectral library but I do not find any other information that could allow me to find the solution. 

Here is a link to a OneDrive folder containing the zip file : https://uclouvain-my.sharepoint.com/:f:/g/personal/maxime_troiani_uclouvain_be/EpI3sBpCZrFIh6m9-HoRA00B1QKdx3vXNxcrPWkDhQcMuw?e=lg973B

Any recommandation would be helpful ! 

Maxime

**Comment by janzandr** [comment]
Please show the error traceback and/or some screenshots explaining how to reproduce the problem.

**Comment by MaximeTroiani** [comment]
Hi, 

Here are some printscreens. I noticed that the error is different if a VRT or a TIF file is used as input. 

![bug_enmapbox_2](https://github.com/EnMAP-Box/enmap-box/assets/155750840/d8cc8d21-95c2-4e37-b132-df8f2cf6274d)
![bug_enmapbox](https://github.com/EnMAP-Box/enmap-box/assets/155750840/eeb3df7b-d04d-4f1b-952b-b902ecc3236e)
![bug_enmapbox_3](https://github.com/EnMAP-Box/enmap-box/assets/155750840/cb8a508e-fd5c-4bf8-8dd0-e4c5092844fc)

**Comment by janzandr** [comment]
Thanks. Yes, that is a known and already fixed bug, related to: https://github.com/EnMAP-Box/enmap-box/issues/786

**Comment by janzandr** [comment]
FYI: I'll release a DEV version with the fix later this week.
